src/csharp/Microsoft.Spark/Sql/Types/DataType.cs

```diff
@@ -56,10 +55,10 @@ public string Json
         {
             get
             {
-                object jObject = (JsonValue is JObject) ?
-                    ((JObject)JsonValue).SortProperties() :
+                object jObject = (JsonValue is JsonElement) ?
```

Comment:
Looking at this some more, `JsonValue` will never be a `JsonElement`. If you look at the complex types definition of JsonValue, they now all return anonymous types, which will never be `JsonElement`.